### ISSUE TITLE
Improve "cache outdated" message

### ISSUE THREAD

**Issue by tooomm** [lycheeverse/lychee#756]
The message contains a huge and highly detailed timestamp currently, with ms and even µs and ns!
`Cache is too old (age: 9days 23h 47m 11s 823ms 371us 801ns, max age: 1d). Discarding`

I suggest to reduce this information to a useful amount, at least the printed version:
`Cache is too old (age: 9days 23h 47m 11s, max age: 1d). Discarding`

It would make this part of the output more human friendly and better read- and usable.

**Comment by lebensterben** [comment]
It can be even shorter.

For example, if the threshold is X days, then it only needs to report up to hours. If the threshold is X hours, then it reports up to minutes.

**Comment by tooomm** [comment]
Removing ms, µs and ns from the output is a simple improvement.

Your's is even more advanced, sure!
`days` should maybe also use the same short form `d` as already used for `max-cache-age`.

**Comment by mre** [comment]
Nice idea. I was just too lazy to print a more human-readable format. If someone wants to give it a shot I'd be thankful for a PR.

**Comment by mre** [comment]
Hm, might not be that easy.
humantime has an [open issue for formatting options](https://github.com/tailhook/humantime/issues/24), but it [doesn't seem to be actively maintained](https://github.com/tailhook/humantime/issues/31).

That leaves us with a few options:
* Find an alternative formatter. Haven't found any good options, though; open for suggestions.
* Maintain the formatting logic ourselves (e.g. by forking humantime or copying over the logic).
* Add `chono` as a dependency, which [supports formatting](https://docs.rs/chrono/latest/chrono/format/index.html).
  (Note that `std::time::Duration` doesn't have a `strfmt`-like interface.)

None of these options sound very promising.
`chono` sounds like the most reasonable option, but I'm not sure if we should add that dependency just for cache age formatting.

If we are willing to accept a fixed format, we could also go with some hacky code like this:

```rust
format!(
    "{}d {}h {}m {}s",
    elapsed.as_secs() / 86400,
    (elapsed.as_secs() % 86400) / 3600,
    (elapsed.as_secs() % 3600) / 60,
    elapsed.as_secs() % 60
)
```

Not keen to add that either.

So...  any ideas? 🤷

**Comment by lebensterben** [comment]
make a wrapper struct on top of the return value of elapsed.as_secs(), and we can implement std::fmt::Format ourselves.

**Comment by mre** [comment]
This was fixed in #849, which made it into `master` now. Guess we can close this. 🥳
